CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+## 1.29.1
+* Feature - Update task cleanup wait logic to clean task resources immediately instead of waiting 3 hours [2084](https://github.com/aws/amazon-ecs-agent/pull/2084)
```

Comment:
this should be enhancement i think. otherwise you will need major rather than minor version bump